2023/debugging/golang/Makefile

```diff
@@ -0,0 +1,29 @@
+reset:
+	rm -f ./**/*_test.go go.mod go.sum
+.PHONY: reset
```

Comment:
I just read through the changes, i did not check if the Makefile and code is actually working and Symfloewr is providing the output you want. Please double-check @bauersimon yourself.